index.html

```diff
@@ -846,237 +846,183 @@ <h4>
         </li>
         <li>
 
-            <p data-lang="en">There are two traditional size systems for Chinese characters, an old one and a new one. The following tables shows the equivalence of the old size system and the new size system in the Western point system respectively.</p>
-            <table class="charsize" id="oldsize-en" data-lang="en">
+            <p data-lang="en">There have been different size systems for Chinese characters. The size system in traditional metal type utilized hào (literally No. ) units, while in the phototypesetting era, <a href="https://www.w3.org/TR/jlreq/#elements_of_kihonhanmen">Q</a> were used as the sizing units instead. When it came to desktop publishing, font sizes were determined by the DTP point system which was built into the software itself. Currently, the traditional hào-system are still used for typesetting in many Chinese publications.</p>
+            <p data-lang="en">These hào-system were not standardized by the various foundries in the past. In addition, point-systems were also different in Anglo-Americas, Europe Continental, DTP and so on, which resulted in numerous conversion methods between hào-system and point-system. The following table lists their most common corresponding conversions as a reference. It is not normative information.</p>
```

Comment:
Some more:

This hao system was ...

"Anglo-Americas, Europe Continental" sounds odd, but maybe those are technical terms. Seems like adjectives without a noun though ('systems'?).

hth

Reply:
yes, these 2 are technical terms, should be as adjectives, for the noun "point(s)".
And, there is a little typo, not "Anglo-Americas" but "Anglo-American" 
I will fix it.

Reply:
Edited.